src/Analyser/ResultCache/ResultCacheManager.php

```diff
@@ -92,21 +92,21 @@ public function restore(array $allAnalysedFiles, bool $debug, bool $onlyFiles, ?
 	{
 		$startTime = microtime(true);
 		if ($debug) {
-			if ($output->isDebug()) {
+			if ($output->isVerbose()) {
```

Comment:
I'd switch to `isVeryVerbose` (it's part of the interface already)

Reply:
Adjusted